tests/import-name/ignored-list-fail/test.ts.lint

```diff
@@ -0,0 +1,7 @@
+import pkg from "fs/package-name";
+import abc from "abc-tag";
+import pqr from "my-module"
+import what from "what-module"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Misnamed import. Import should be named 'whatModule' but found 'what']
```

Comment:
Sorry @IllusionMH I don't follow?

Reply:
I mean that initially test also [included fixes](https://github.com/Microsoft/tslint-microsoft-contrib/blob/71cf21d897e7c30e8ae2bf141ed37375d3f1ac65/src/tests/ImportNameRuleTests.ts#L218) which aren't present in new format.
However I'm not sure if they are really needed in this folder because fixes were already tested in `default` folder.

Reply:
Oh sure. I'll add them back in anyway, seems nice to have.